style.css

```diff
@@ -0,0 +1,71 @@
+.menu {
+  display: flex;
+  justify-content: right;
+  padding: 20px;
+}
+
+.page {
+  display: flex;
+  flex-direction: column;
+  margin: unset;
+  overflow: auto;
+}
+
+.greetings {
+  line-height: 52px;
+  font-weight: 700px;
+  justify-content: center;
+  font-size: 40px;
+  margin-top: unset;
+  margin-bottom: unset;
+  color: #172b4d;
+}
+
+.job {
+  color: #36b37e;
+}
+
+.description {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #172b4d;
+}
```

Comment:
As mentioned by the previous reviewer, please update your design to use the correct font family for the main title and description. I believe the correct font family is `Inter`. You should be able to search for this font on [google fonts](https://fonts.google.com/) and add it to your project🙏